packages/components/codemods/README.md

```diff
@@ -25,6 +25,12 @@ pnpm kaizen-codemod src migrateWellVariantToColor
 
 ## Available codemods
 
+Released in `1.80.5`
```

Comment:
Might want to change this to 1.80.6

Reply:
Many thanks :) update this and added those other changes so should be good for a final review 🤞